neonvm-daemon/pkg/cpuscaling/sys_fs_state_scaling.go

```diff
@@ -0,0 +1,196 @@
+package cpuscaling
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+// CPU directory path
+const cpuPath = "/sys/devices/system/cpu/"
+
+type CPUSysFsStateScaler struct {
+}
```

Comment:
What's the intent behind making an empty type with methods, instead of just public functions?

Reply:
Honestly, not a lot of reason to have it struct, but I like less coupled code in the wrapping cpuServer structure much more than just a calls to public functions. 

Plus I was originally going to add unittests here and forget :D